Makefile

```diff
@@ -23,6 +23,7 @@ build-deps:
 
 deps: build-deps
 	govendor sync -v
+	git checkout vendor/github.com/brocaar/lorawan/band/band_au915_928.go
```

Comment:
Won't this blow up if types in `github.com/brocaar/lorawan/band` go through breaking changes? Maybe we can specify a commit, or a tag?

Reply:
That package is tagged by govendor, we simply override it with a checked-in version.